tests/unit_tests/test_db_handler.py

```diff
@@ -316,23 +316,37 @@ def test_separating_get_and_write(db):
         assert io.getOutputFile(fileNow, dirType="model", test=True).exists()
 
 
-def test_insert_files_db(db):
+def test_insert_and_get_files_db(db):
 
     logger.info("----Testing inserting files to the DB-----")
     logger.info(
         "Creating a temporary file in {}".format(io.getOutputDirectory(dirType="model", test=True))
     )
-    fileName = io.getOutputDirectory(dirType="model", test=True) / "test_file.dat"
-    with open(fileName, "w") as f:
-        f.write("# This is a test file")
+    outputDir = io.getOutputDirectory(dirType="model", test=True)
+    fileNames = ["test_file.dat", "test_file2.dat"]
+    for file in fileNames:
+        with open(outputDir / file, "w") as f:
+            f.write("# This is a test file")
+        file_id = db.insertFileToDB(outputDir / file, "sandbox")
+        assert file_id == db._getFileMongoDB("sandbox", file)._id
 
-    file_id = db.insertFileToDB(fileName, "sandbox")
-    assert file_id == db._getFileMongoDB("sandbox", "test_file.dat")._id
+    logger.debug("Getting files from DB to {}".format(outputDir))
+    db.exportFilesDB("sandbox", outputDir, fileNames)
+
+    logger.debug("Checking if files were written to {}".format(outputDir))
+    for file in fileNames:
+        assert io.getOutputFile(outputDir / file, dirType="model", test=True).exists()
```

Comment:
This test was not in the original function, but I think it's a good one to have, so please keep this line (just not in a for loop).